### PR TITLE
Add vmblog about CRI-O to awesome list

### PR DESCRIPTION
**Pull request by saschagrunert**
The blog post has been release recently and should be part of our list.

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
LGTM

File: awesome.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ collects useful resources about CRI-O.
 
 ### Articles
 
+- [Under the hood of CRI-O](https://vmblog.com/archive/2019/10/30/under-the-hood-of-cri-o-kubernetes-container-runtime.aspx)
```
Comment (vrothberg) [inline awesome.md]: The date is missing

Comment (saschagrunert) [reply awesome.md]: Thanks for the hint, I added the date.

**Pull request merged**